src/components/Item/Item.js

```diff
@@ -27,4 +28,9 @@ const Item = ({ value, isDone, onClickDone, id, onClickDelete }) => (
         </ListItemSecondaryAction>
     </ListItem>);
 
+Item.propTypes = {
+    value: propTypes.string,
```

Comment:
Codacy found an issue: ['propTypes' is not defined.](https://app.codacy.com/gh/WebHeroSchool/ReactProject_karolina/pullRequest?prid=6064351)